src/lib/routes/admin-api/user-admin.ts

```diff
@@ -123,6 +137,50 @@ export default class UserAdminController extends Controller {
             ],
         });
 
+        this.route({
+            method: 'get',
+            path: '/:id/pat',
+            handler: this.getUserPats,
+            permission: READ_USER_PAT,
+            middleware: [
+                openApiService.validPath({
+                    tags: ['Users'],
+                    operationId: 'getUserPats',
+                    responses: { 200: createResponseSchema('patsSchema') },
+                }),
+            ],
+        });
+
+        this.route({
+            method: 'post',
+            path: '/:id/pat',
+            handler: this.createUserPat,
+            permission: CREATE_USER_PAT,
+            middleware: [
+                openApiService.validPath({
+                    tags: ['Users'],
+                    operationId: 'createUserPat',
+                    requestBody: createRequestSchema('patSchema'),
+                    responses: { 200: createResponseSchema('patSchema') },
+                }),
+            ],
+        });
+
+        this.route({
+            method: 'delete',
+            path: '/:id/pat/:patId',
+            acceptAnyContentType: true,
```

Comment:
Any specific reason for this?

Reply:
I think you get invalid content type if not defining it, which really does not make sense for a delete request with empty body. Would maybe be better to not require a content type for delete requests in general (that would be outside of this PR)?